src/components/InputSelect/index.tsx

```diff
@@ -0,0 +1,54 @@
+import clsx from 'clsx';
+import React, { ChangeEvent } from 'react';
+
+interface Props {
+  className?: string;
+  id: string;
+  label: string;
+  onChange: (value: string) => void;
+  options: { label: string; value: string }[];
+  placeholder?: string;
+  required?: boolean;
+  value: string;
+}
+
+const InputSelect: React.FC<Props> = ({
+  className,
+  id,
+  label,
+  onChange,
+  options,
+  placeholder,
+  required,
+  value
+}) => {
+  const handleChange = (event: ChangeEvent<HTMLSelectElement>) => {
```

Comment:
This method makes things a bit harder to use. As a developer when I use your input `InputSelect` and I add a classic `onChange={handleChange}` I would expect that on each `onChange` I will receive an event object. Why the need to abstract this? It seems that this wrapper only open the gates for:

- All input abstracted need a function to make sure we pass value instead `event`
- You need to know there is no event object being passed to the `onChange` function